app/models/announcement.rb

```diff
@@ -8,11 +8,21 @@
 #  title        :string
 #  created_at   :datetime         not null
 #  updated_at   :datetime         not null
+#  user_id      :bigint
```

Comment:
It looks like lines 11-19 are a new addition. Are they providing value?

Reply:
They were added with the migration and might be useful for future interns. It's the schema information and follows the same style as some of the other files.